build/cpp/cmd/src/cmd.cc

```diff
@@ -186,15 +192,15 @@ void cmd::CommandLine::Profile(const std::string &username) {
   const auto &result = func_client_->event(4, any);
   result->UnpackTo(&reply);
   std::cout << "Followers: ";
-  for (int i = 0; i < reply.followers().size(); i++) {
+  for (int i = 1; i < reply.followers().size(); i++) {
```

Comment:
fixed a bug unrelated to ephemeral code, so a user is not shown as a follower and following of themself